Makefile

```diff
@@ -67,6 +67,6 @@ cross: $(TOOLS_BINDIR)/makefat
 test-companion:
 	GOOS=linux go build -ldflags "$(LDFLAGS)" -o bin/test-companion ./cmd/test-companion
 
-.PHONY: test
-test: gvproxy test-companion
-	go test -timeout 20m -v ./...
+.PHONY: test-linux
+test-linux: gvproxy test-companion
+	go test -timeout 20m  -v ./test-qemu
```

Comment:
`test-linux` is not running as many tests as what `test` was running, `go test ./...` was running all of these:
```
pkg/services/dns/dns_test.go
pkg/services/dns/hosts_file_test.go
pkg/tap/ip_pool_test.go
test-win-sshproxy/basic_test.go
test-win-sshproxy/suite_test.go
test/basic_test.go
test/efi_darwin_arm64_test.go
test/port_forwarding_test.go
test/suite_test.go
```
Imo we can keep a `test` target which runs `go test ./...` but add build tags to the vfkit tests so that they are only run on macos and/or use the `-skip` parameter together with `go test ./...` to have more control over what is being run.